gnome-extra/cinnamon/cinnamon-4.4.8.ebuild

```diff
@@ -0,0 +1,193 @@
+	>=x11-libs/libXfixes-5.0
+	>=x11-wm/muffin-4.0.3[introspection]
+	dev-libs/keybinder:3[introspection]
+	>=x11-libs/libnotify-0.7.3:0=[introspection]
+"
+# Runtime-only deps are probably incomplete and approximate.
+# Each block:
+# 2. Introspection stuff + dconf needed via imports.gi.*
+# 3. gnome-session is needed for gnome-session-quit
+# 4. Control shell settings
+# 5. accountsservice is needed for GdmUserManager (0.6.14 needed for fast
+#    user switching with gdm-3.1.x)
+# 6. caribou needed for on-screen keyboard
+# 7. xdg-utils needed for xdg-open, used by extension tool
+# 8. imaging, lxml needed for cinnamon-settings
+# 9. gnome-icon-theme-symbolic needed for various icons
+# 10. pygobject needed for menu editor
+# 11. nemo - default file manager, tightly integrated with cinnamon
+# 12. polkit-gnome - explicitly autostarted by us
+# TODO(lxnay): fix error: libgnome-desktop/gnome-rr-labeler.h: No such file or directory
+RDEPEND="${COMMON_DEPEND}
+	>=gnome-base/dconf-0.4.1
+	>=gnome-base/libgnomekbd-2.91.4[introspection]
+	sys-power/upower[introspection]
+
+	>=gnome-extra/cinnamon-session-4.0
+	>=gnome-extra/cinnamon-settings-daemon-4.0
+
+	>=app-accessibility/caribou-0.3
+
+	dev-libs/libtimezonemap
+	x11-misc/xdg-utils
+	x11-libs/xapps[introspection]
+
+	$(python_gen_cond_dep '
+		dev-python/dbus-python[${PYTHON_USEDEP}]
+		dev-python/pygobject:3[${PYTHON_USEDEP}]
+		dev-python/pexpect[${PYTHON_USEDEP}]
+		dev-python/pycairo[${PYTHON_USEDEP}]
+		dev-python/pyinotify[${PYTHON_USEDEP}]
+		dev-python/pypam[${PYTHON_USEDEP}]
+		dev-python/pillow[${PYTHON_USEDEP}]
+		dev-python/setproctitle[${PYTHON_USEDEP}]
+		dev-python/tinycss[${PYTHON_USEDEP}]
+		dev-python/pytz[${PYTHON_USEDEP}]
+		dev-python/xapp[${PYTHON_USEDEP}]
+	')
+
+	x11-themes/gnome-themes-standard
+	x11-themes/adwaita-icon-theme
+
+	>=gnome-extra/nemo-4.0
+	>=gnome-extra/cinnamon-control-center-4.0[networkmanager=]
+	>=gnome-extra/cinnamon-screensaver-4.0
+
+	gnome-extra/polkit-gnome
+
+	nls? ( >=gnome-extra/cinnamon-translations-4.0 )
+"
+DEPEND="${COMMON_DEPEND}
+	dev-util/gtk-doc
+	>=dev-util/intltool-0.4
+	>=sys-devel/gettext-0.17
+	virtual/pkgconfig
+	!!=dev-lang/spidermonkey-1.8.2*
+"
+# libmozjs.so is picked up from /usr/lib while compiling, so block at build-time
+# https://bugs.gentoo.org/show_bug.cgi?id=360413
+
+S="${WORKDIR}/cinnamon-${PV}"
+
+src_prepare() {
+	# Fix backgrounds path as cinnamon doesn't provide them
+	# https://github.com/linuxmint/Cinnamon/issues/3575
+	eapply "${FILESDIR}"/${PN}-3.8.0-gnome-background-compatibility.patch
+
+	# Use wheel group instead of sudo (from Fedora/Arch)
+	# https://github.com/linuxmint/Cinnamon/issues/3576
+	eapply "${FILESDIR}"/${PN}-3.6.6-wheel-sudo.patch
+
+	# Add polkit agent to required components (from Fedora/Arch), bug #523958
+	# https://github.com/linuxmint/Cinnamon/issues/3579
+	sed -i 's/RequiredComponents=\(.*\)$/RequiredComponents=\1polkit-gnome-authentication-agent-1;/' \
+		files/cinnamon*.session.in || die
```

Comment:
Is that actually necessary, given we install XDG autostart entry here:
https://github.com/gentoo/gentoo/pull/15774/files#diff-204de02dcc66a5092970bb1634e2e9efR165-R168?

Reply:
I presume this is enforcing that the polkit daemon is started before the cinnamon session. Seems necessary to ensure proper start-up ordering.

Reply:
Hmm, okay, that requires further investigation, let's leave it for now and I will have a look into this when I got a minute and we can address separately.